homeassistant/components/aussie_broadband/sensor.py

```diff
@@ -123,27 +138,24 @@ class AussieBroadandSensorEntity(CoordinatorEntity, SensorEntity):
     """Base class for Aussie Broadband metric sensors."""
 
     def __init__(
-        self, service: dict[str, Any], description: SensorEntityDescription
+        self, service: dict[str, Any], description: SensorValueEntityDescription
     ) -> None:
         """Initialize the sensor."""
         super().__init__(service["coordinator"])
-        self.entity_description = description
+        self.entity_description: SensorValueEntityDescription = description
```

Comment:
We can move the type annotation to a class attribute above the init method.
```suggestion
        self.entity_description = description
```